utils/context/AuthProvider.js

```diff
@@ -19,13 +20,26 @@ export function AuthProvider({ children }) {
   const [user, loading, error] = useAuthState(auth);
   const router = useRouter();
 
-  // console.log(user);
-
-  const signup = (email, password, username) => {
-     createUserWithEmailAndPassword(auth, email, password)
+  const signup = (body) => {
+     createUserWithEmailAndPassword(auth, body.email, body.password)
       .then((response) => {
+         body['uid'] = response.user.uid
+        if (body.account_type === 'Mentor') {
+          axios.post('/pages/mentors/index.js', body)
+            .then(res => {
+              console.log(res)
+            }).catch(err => {
+              console.error(err.message)
+            })
+        } else {
+          axios.post('/pages/mentees/index.js', body)
```

Comment:
Did you mean to send these to this route? Thought api route was `/api/*`. Not sure but lmk if that was working properly